wagtail/test/testapp/models.py

```diff
@@ -1011,6 +1011,28 @@ class Meta:
 register_snippet(AdvertWithTabbedInterface)
 
 
+class AdvertManager(models.Manager):
+    pass
+
+
+class AdvertWithCustomManager(models.Model):
```

Comment:
Thanks, but we don't need a new model for this, let's just reuse an existing one. Adding a new model would add additional overhead as that's yet another migration to run and another table to create on the database.

Reply:
Will do :+1:

Reply:
Wouldn't that create a new migration in any case? Adding a new manager to an existing model will need a migration as well, won't it?

Reply:
You're right, yep, but it won't create a new database table.

Reply:
I just tried here locally with the `Advert` model (the simplest one), and besides creating the migration I would also have to change all occurrences of `Advert.objects.foo` in the tests (around 70).

The one with the least amount of test occurrences (2) is `AdvertWithCustomUUIDPrimaryKey`, but I think putting a custom manager on this one hurts the readability of tests.

Please let me know how you'd like to proceed

Reply:
My bad, yep. We can go with a simple model without any fields (other than the primary key), and name it something like `CustomManagerModel` or `ModelWithCustomManager` I suppose.